plugin/pkg/admission/initialresources/admission_test.go

```diff
@@ -110,7 +110,7 @@ func expectNoAnnotation(t *testing.T, pod *api.Pod) {
 	}
 }
 
-func admit(t *testing.T, ir admission.Interface, pods []*api.Pod) {
+func admit(t *testing.T, ir admission.MutationInterface, pods []*api.Pod) {
```

Comment:
s/Mutation/Revising/g (I think that is the term we will end up going with)

Reply:
> s/Mutation/Revising/g (I think that is the term we will end up going with)

We starting talking about this as mutation and wrote docs about mutation.  I don't remember discussing a name change.  I don't find "revising" to be a natural word.  If we do decide on a name change after this is in, the newer golang tools allow doing it without much pain.